DigitalLearningSolutions.Web/Helpers/TrackingSystem/SelectedPage.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace DigitalLearningSolutions.Web.Helpers.TrackingSystem
+{
+    public enum SelectedPage

```

Comment:
Should this be called "DelegatePage"?

Reply:
I couldn't decide between having a specific enum for each sub section of the tracking system (e.g. DelegatePage, CentrePage, etc.) or just a single enum that contains all pages in one. Do we have a preference?

Reply:
Maybe use delegate specific for now so that's the only thing that can be passed into the delegate side menu partial. And if we decide to commonise some of this later, we can change it 🙏